readme.md

```diff
@@ -986,7 +986,7 @@ const h2got = got.extend({request});
 | Browser support       |       ✖      |       ✖      |       ✔*     |       ✔      |
 | Electron support      |       ✔      |       ✖      |       ✖      |       ✖      |
 | Promise API           |       ✔      |       ✔      |       ✔      |       ✔      |
-| Stream API            |       ✔      |       ✔      |       ✖      |       ✖      |
+| Stream API            |       ✔      |       ✔      | Node.js only |       ✖      |
```

Comment:
maybe `Node-specific`?

Reply:
I went with something that is consistent with the existing "Browser only".